src/cluster-configuration/deploy/stop.sh.template

```diff
@@ -31,8 +31,17 @@ if kubectl get configmap | grep -q "gpu-configuration"; then
     kubectl delete configmap gpu-configuration || exit $?
 fi
 
+if kubectl get configmap | grep -q "pai-version"; then
+    kubectl delete configmap pai-version || exit $?
+fi
+
+if kubectl get configmap | grep -q "k8s-version"; then
+    kubectl delete configmap k8s-version || exit $?
```

Comment:
kubernetes-configuration.yaml has this information.

Reply:
There is slight difference: the k8s version is our 'customized k8s' version.
For example: v0.11.0-1.9.9 may different from v0.12.0-1.9.9, although they both basing on the k8s 1.9.9.